### PR TITLE
initial_commit_activejob_web

### PR DESCRIPTION
**Pull request by MohammedNazeer10**

### CODE REVIEW

**Review by rajeshkanna-mallow — APPROVED** [review]

**Review by gomathi-mallow — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# Specify your gem's dependencies in activejob-web.gemspec.
+gemspec
+
+gem "puma"
+
+gem 'pg'
+
+gem "sqlite3"
```
Comment (rajeshkanna-mallow) [inline Gemfile:11]: @MohammedNazeer10 As we decided to gow with pg, So why we are still having this sqlite3 gem in Gemfile?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/.bundle/
+/doc/
+/log/*.log
+/pkg/
+/tmp/
+/test/dummy/db/*.sqlite3
+/test/dummy/db/*.sqlite3-*
+/test/dummy/log/*.log
+/test/dummy/storage/
+/test/dummy/tmp/
```
Comment (rajeshkanna-mallow) [inline .gitignore:10]: @MohammedNazeer10  What is the purpose of having dummy folder?

Comment (MohammedNazeer10) [reply .gitignore:10]: @rajeshkanna-mallow , it is created by default in rails engine for the purpose of testing and verifying the functionality.

File: Gemfile (hunk 1 above)
Comment (MohammedNazeer10) [reply Gemfile]: @rajeshkanna-mallow , i have removed and updated PR

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/.bundle/
+/doc/
+/log/*.log
+/pkg/
+/tmp/
+/test/dummy/db/*.sqlite3
+/test/dummy/db/*.sqlite3-*
```
Comment (rajeshkanna-mallow) [inline .gitignore:7]: Can we remove sqlite related code changes from here and add if any changes required for pg? @MohammedNazeer10

File: test/dummy/config/database.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
```
Comment (rajeshkanna-mallow) [inline test/dummy/config/database.yml:6]: Please update these comments as per pg related configuration @MohammedNazeer10

File: README.md [hunk 5]
```diff
@@ -1,2 +1,28 @@
-# activejob-web
```
Comment (rajeshkanna-mallow) [inline README.md:1]: Can you update readme file? @MohammedNazeer10

File: test/dummy/config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: activejob_web_test
+
+production:
+  <<: *default
+  database: activejob_web_production
```
[the remark this answers is not in the document]
Comment (MohammedNazeer10) [reply test/dummy/config/database.yml:25]: yes @rajeshkanna-mallow

**Comment by gomathi-mallow** [comment]
> @gomathi-mallow I've done review with this PR, can I merge it?

Yes, Fine with me @rajeshkanna-mallow

Note: We need to include RSpec for testing. We shall include it later

**Comment by rajeshkanna-mallow** [comment]
@MohammedNazeer10 Can you get approval from @JPMallow ?

**Pull request merged**